src/pages/sangil/Main/Main.js

```diff
@@ -0,0 +1,63 @@
+// 항상 시작전에 git checkout feature/si 실시하고 확인할것
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Main_page from './Main_page';
+import Nav from './Nav';
+import 이미지 from '../../../assets/images/profile_si.png';
+import '../Main/Main.scss';
+
+function Main() {
+  return (
+    <div>
+      <Nav />
+      <Main_page />
+      <div className="main_right">
+        <div className="user_profile">
+          <div className="profile_small_img">
+            <img src={이미지} alt="프로필이미지" />
```

Comment:
```suggestion
            <img src={이미지} alt="프로필이미지" />
```

오 이렇게 이미지를 미리 import 해주고 사용할 수 있군요!! 엄청 직관적이네요 짱짱